### PR TITLE
- updated the ArcGIS Runtime SDK for Android version to 100.6.0

### PR DESCRIPTION
**Pull request by puneet-pdx**
- added RELEASE.md

for issue: https://github.com/Esri/maps-app-android/issues/176

### CODE REVIEW

**Comment by puneet-pdx** [comment]
@LDuncAndroid @gunt0001 can either of you review this PR. The main change is to just update this sample so that it works with the latest version of ArcGISRuntime. 

I have made two additional changes,
- removed an unused string variable
- moved the version numbers to the project build.gradle file to make this sample consistent with the other two released samples.

File: build.gradle [hunk 1]
```diff
@@ -18,6 +18,17 @@ allprojects {
 }
 // Define versions in a single place
 ext {
+    // Sdk and tools
+    minSdkVersion = 19
+    targetSdkVersion = 28
+    compileSdkVersion = 28
```
Comment (gunt0001) [inline build.gradle]: Probably worth to upgrade target and compile SDK versions to 29

File: build.gradle [hunk 2]
```diff
@@ -18,6 +18,17 @@ allprojects {
...
+
+    // App dependencies
+    supportLibraryVersion = '28.0.0'
```
Comment (gunt0001) [inline build.gradle]: 29.0.0?

File: maps-app/build.gradle [hunk 3]
```diff
@@ -1,13 +1,13 @@
...
 android {
-    compileSdkVersion 28
+    compileSdkVersion rootProject.ext.compileSdkVersion
 
 
     defaultConfig {
         applicationId "com.esri.android.mapsapp"
-        minSdkVersion 19
-        targetSdkVersion 28
+        minSdkVersion rootProject.ext.minSdkVersion
```
Comment (gunt0001) [inline maps-app/build.gradle]: `ext` redundant? -> `rootProject.minSdkVersion`

File: maps-app/build.gradle [hunk 4]
```diff
@@ -1,13 +1,13 @@
...
+        targetSdkVersion rootProject.ext.targetSdkVersion
```
Comment (gunt0001) [inline maps-app/build.gradle]: `ext` redundant? -> `rootProject.targetSdkVersion`

File: build.gradle (hunk 1 above)
Comment (puneet-pdx) [reply build.gradle]: updated

File: build.gradle (hunk 2 above)
Comment (puneet-pdx) [reply build.gradle]: the latest supportLibraryVersion available is 28.0.0

File: maps-app/build.gradle (hunk 3 above)
Comment (puneet-pdx) [reply maps-app/build.gradle]: updated

File: maps-app/build.gradle (hunk 4 above)
Comment (puneet-pdx) [reply maps-app/build.gradle]: updated

**Comment by puneet-pdx** [comment]
@gunt0001 I have update the target and compiler sdk versions(there is an error in the commit message stating min sdk version) . The latest support lib version out is 28.0.0

**Comment by puneet-pdx** [comment]
@esreli can you please review.

**Comment by esreli** [comment]
(assuming someone else has vetted the environment changes)

**Pull request merged**